app/src/test/java/com/android/unio/model/association/AssociationRepositoryFirestoreTest.kt

```diff
@@ -115,12 +118,23 @@ class AssociationRepositoryFirestoreTest {
             "fullName" to association1.fullName,
             "category" to association1.category.name,
             "description" to association1.description,
-            "members" to association1.members.uids,
+            "members" to
+                mapOf(
+                    "1" to "Guest",
+                    "2" to "Guest"), // the serialization process does not allow us to simply put
+            // association1.members
+            "roles" to
+                mapOf(
+                    "Guest" to
+                        mapOf("displayName" to "Guest", "permissions" to listOf("Full rights")),
+                    "Administrator" to
+                        mapOf(
+                            "displayName" to "Administrator",
+                            "permissions" to listOf("Full rights"))),
             "followersCount" to association1.followersCount,
             "image" to association1.image,
             "events" to association1.events.uids,
-            "principalEmailAddress" to association1.principalEmailAddress,
-            "adminUid" to association1.adminUid)
+            "principalEmailAddress" to association1.principalEmailAddress)
```

Comment:
even though these are tests, changing the key strings to the form Association::parameter.title would make the test work even if we change the name of a parameter is changed